src/react-bubble-chart-d3.js

```diff
@@ -312,4 +320,6 @@ BubbleChart.defaultProps = {
     color: '#fff',
     weight: 'normal',
   },
+  bubbleClickFun: (node) => {console.log(`Bubble ${node.label} is clicked ...`, n)},
```

Comment:
here you are trying to access to the node.label and in the call of the function you are passing the label directly here should be ```bubbleClickFun: (label) => {console.log(`Bubble ${label} is clicked ...`, n)},``` same for legendClickFun

Reply:
Yes, This would be good.